webpack.config.js

```diff
@@ -0,0 +1,57 @@
+const config = require('./package.json').config
+const path = require('path')
+
+const webpack = require('webpack')
+
+module.exports = function(env = {}) {
+  const plugins = {
+    moduleConcatenation: new webpack.optimize.ModuleConcatenationPlugin(),
+    uglify: new webpack.optimize.UglifyJsPlugin({
+      output: { comments: false },
+      test: /min.js$/i,
```

Comment:
did this because of doing only on a minified file as described on our `entry` attr